app/assets/javascripts/models/beta/iteration.js

```diff
@@ -198,10 +230,23 @@ export const groupBySprints = (stories = [], project, initialSprintNumber = 1, p
       return addStoryToSprint(project, sprints, firstSprintIndex, story, velocity);
     }
 
+    if (canEnterOnLastSprint(sprints, story, velocity)) {
```

Comment:
What do you think about renaming it to `canEnterLastAddedSprint` or `willFitLastAddedSprint` to reflect that the _last_ is not exactly the last of all sprints but the last added one until now?